docs/docs/30-how-to-guides/14-working-with-stage.md

```diff
@@ -0,0 +1,42 @@
+---
+description: Learn how to work effectively with stages
+sidebar_label: Working with stage
+---
+
+# Working With Stage
```

Comment:
```suggestion
# Working With Stages
```

Reply:
See [this comment](https://github.com/akuity/kargo/pull/2696/files#r1819141803) on another PR.

This aligns nicely with what I was suggesting there.

Doubling down on that, I think there are parts of the concepts doc that should move to this new page.

Reply:
No. Grammatically, "Freight" is a [mass noun](https://en.wikipedia.org/wiki/Mass_noun). This is why you may often see phrases like "a piece of Freight" or "two pieces of Freight" to add units where none are otherwise specified.

Not everyone is aware of this, so you'll doubtlessly encounter places in the code or docs where "Freights" is used, but it is grammatically incorrect. It may also appear in generated code because the tools in question don't understand mass nouns.

As far as documentation is concerned, we should strive for grammatical correctness when we can.